sass/_variables.scss

```diff
@@ -0,0 +1,16 @@
+// CRM Related Variables
+$crm-input-background-dark:               #f3f3f6;
+$crm-input-border:                        #c2cfd8;
+
+// Mosaico Related Variables
+$crm-mosaico-template-item:               #f1f5f6;
+$crm-mosaico-template-border:             #e8eef0;
+$crm-mosaico-template-item-wrapper:       #333;
+$crm-mosaico-template-item-shadow:        inset 0 1px 10px 0 rgba(0, 0, 0, .25);
+
+// Identity Colors
+$brand-primary:                           #42AFCB;
```

Comment:
Its required by this file 
https://github.com/compucorp/uk.co.vedaconsulting.mosaico/pull/4/files#diff-a9e67b892cd069c8d017d3e8687d28f4

I get "Variable not defined" error if i removed it from _variables.scss